dev-client/package.json

```diff
@@ -44,7 +44,7 @@
     "react-native-svg": "^14.0.0",
     "react-native-tab-view": "^3.5.2",
     "react-native-vector-icons": "^10.0.2",
-    "terraso-client-shared": "github:techmatters/terraso-client-shared#23fb61c",
+    "terraso-client-shared": "github:techmatters/terraso-client-shared#david-dev-branch",
```

Comment:
issue: please use a description branch name

Reply:
I don't understand this comment.